### PR TITLE
Added New React Project and Created section for Android ideas

### PR DESCRIPTION
**Pull request by danishsshaikh**
Thanks for this ♥

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,8 @@ We'll organise into different streams here.
 * Weather App
 * Recipe search App
 * E-commerce app
+* Tinder Clone
```
Comment (praveenscience) [inline README.md:38]: Dei LoL! 🤣

**Review by praveenscience — APPROVED** [review]
👍

**Comment by praveenscience** [comment]
Thanks for your contribution. Optional Request: [Subscribe to my **YouTube Channel**](https://www.youtube.com/praveenscience?sub_confirmation=1), if you haven't already subscribed and share it with your friends too.

**Comment by danishsshaikh** [comment]
> Thanks for your contribution. Optional Request: [Subscribe to my **YouTube Channel**](https://www.youtube.com/praveenscience?sub_confirmation=1), if you haven't already subscribed and share it with your friends too.

Just did 👍

File: README.md (hunk 1 above)
Comment (praveenscience) [reply README.md:38]: @danishsshaikh Resolve this conversation once you have seen this! 🤣

Comment (danishsshaikh) [reply README.md:38]: haha, got it xD

**Pull request merged**